core/include/detray/materials/material_rod.hpp

```diff
@@ -0,0 +1,41 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Detray include(s)
+#include "detray/definitions/qualifiers.hpp"
+
+namespace detray {
+
+// Rod structure to be mapped on the mask
```

Comment:
I don't fully understand this class, is this supposed to be used for e.g. describing a cylinder ?

Reply:
I was planning to use it for the wires. Cylinders would take a slab.
But It is also possible that I end up with not using this class depending on the future development - so let's just leave it here for now until I clearly figure out how to handle the wire geometry.